modules/sb-install-service-catalog.adoc

```diff
@@ -5,7 +5,7 @@
 [id="sb-install-service-catalog-{context}"]
 = Installing service catalog
 
-You must install the service catalog by completing the following steps if you plan on using any of the services from the {asb-name} or template service broker.
+You must install the service catalog by completing the following steps if you plan on using any of the services from the {asb-name} or {tsb-name}.
```

Comment:
If you plan on using any of the services from the {asb-name} or {tsb-name}, you must install the service catalog by completing the following steps.